rts/motoko-rts/src/bigint.rs

```diff
@@ -129,7 +129,7 @@ extern "C" {
     pub(crate) fn bigint_trap() -> !;
 }
 
-#[inline]
+#[inline(always)]
```

Comment:
Probably, `#[inline]` is sufficient (or even not specifying the attribute at all). I measured no real difference, even a very slight degrade...

Reply:
Reverted